fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -89,6 +89,17 @@ object ItemKicker {
     case FreeHtmlKickerWithLink(body, _) => Some(body)
     case _ => None
   }
+
+  /** Return the canonical display text for a kicker */
+  def kickerDisplayText(itemKicker: ItemKicker): Option[String] = itemKicker match {
```

Comment:
In any case, the `asPlainText` would return either a constant string (i.e., `Breaking`) or one of the kicker's data fields, unmodified.  This would assist in keeping the canonical names of the kickers in one place, so they don't diverge across different projects.